src/main/java/in/projecteka/consentmanager/link/link/LinkController.java

```diff
@@ -67,8 +67,11 @@
      * @param patientLinkRequest
      * @return
      */
-    @PostMapping("/v1/links/link/confirm")
-    public Mono<PatientLinkResponse> confirmLink(@RequestBody PatientLinkRequest patientLinkRequest) {
+    @PostMapping("/v1/links/link/confirm/{linkRefNumber}")
```

Comment:
/v1/links/{linkRefNumber}/confirm/

doesn't it make sense?

Reply:
wanted it to keep it same as the older one, so that on android side it will be just replacing of the URL. I agree that the /v1/links/link/{linkRefNumber}/confirm is better. Notice the URL .. because we will have /v1/links/unlink/{linkRefNumber} in future.